content/reference/lattice-protocols/control-interface.md

```diff
@@ -13,15 +13,14 @@ The following is a list of the operations supported by the control interface.
 
 ### NATS Control Interface
 
-All of the control interface messages published on NATS topics use a standard prefix. This prefix is `wasmbus.ctl.{namespace}` where `namespace` is a string used to differentiate one lattice from another. Note that this namespace must correspond to the namespace prefix of the lattice you intend to control. As an example, if your lattice namespace prefix is `prod`, then you would subscribe to the lattice event stream at `wasmbus.ctl.prod.events`.
+All of the control interface messages published on NATS topics use a standard prefix. This prefix is `wasmbus.ctl.{namespace}` where `namespace` is a string used to differentiate one lattice from another. Note that this namespace must correspond to the namespace prefix of the lattice you intend to control.
 
 ⚠️ You need to ensure that your namespace prefix does not contain any characters that cannot appear in a NATS subject.
 
-All control interface messages are serialized as _message pack_ payloads that conform to the schema found in the [actor interfaces](https://github.com/wasmcloud/actor-interfaces) repository.
```

Comment:
This is a comment for the above line:
```
⚠️ You need to ensure that your namespace prefix does not contain any characters that cannot appear in a NATS subject.
```
We don't want to be in the business of keeping a full list of not-allowed things, but could we say something like "Combinations of ASCII letters and numbers are an example of valid characters for NATS subjects" or point out any common mistakes (no trailing spaces, etc)